### PR TITLE
David's SB additions

### PR DESCRIPTION
**Pull request by jordansread**
has #42 in here @wdwatkins

### CODE REVIEW

File: scripts/visualize/visualize-map.R [hunk 1]
```diff
@@ -72,14 +72,18 @@ visualize.states_svg <- function(viz){
 
 process.time_json <- function(viz){
   data <- readDepends(viz)
+  library(dplyr)
   sites <- data[['site-map']]
+  sites.w.data <- readr::read_csv('cache/allSitesYears_355.csv')
```
Comment (jordansread) [inline scripts/visualize/visualize-map.R]: this isn't using the actual data yet

**Comment by jordansread** [comment]
I think this needs to stay open for some clean up. But it has the fix in it.

**Comment by jordansread** [comment]
This is ready for review, but has a small WIP for the bar data `process`

File: scripts/visualize/visualize-map.R (hunk 1 above)
Comment (jordansread) [reply scripts/visualize/visualize-map.R]: fixed now

**Comment by wdwatkins** [comment]
The Virgin Islands site(s) were the problem then?

**Pull request merged**